README.md

```diff
@@ -100,3 +148,6 @@ License &amp; copyright
 Copyright (c) 2014 ghauth contributors (listed above).
 
 ghauth is licensed under the MIT license. All rights not explicitly granted in the MIT license are reserved. See the included LICENSE.md file for more details.
+
+[df]: https://docs.github.com/en/developers/apps/authorizing-oauth-apps#device-flow
```

Comment:
I'd prefer these be inlined like other links in the doc, please

Reply:
No problem.  Fixing.

Reply:
Fixed.